src/MatrixNetwork.jl

```diff
@@ -148,4 +148,37 @@ end
 
 function is_connected(A::SparseMatrixCSC)
     return maximum(scomponents(A).map) == 1
-end
\ No newline at end of file
+end
+
```

Comment:
Combine the documentation into a single string. See preferential_attachment, for instance.